Utilities/CodeGeneration/EdFi.Ods.CodeGen/Mustache/Entities_PropertyAccessors.mustache

```diff
@@ -68,6 +68,11 @@
             }
             set
             {
+                {{#UsiProperty}}
```

Comment:
This entire template section is in an #UsiProperty check, so no need to verify it again here.  Just add the new if statement.

Reply:
Done